wizwalker/memory/memory_objects/window.py

```diff
@@ -346,7 +348,7 @@ async def card_size_horizontal(self) -> int:
     async def card_size_vertical(self) -> int:
         return await self.read_value_from_offset(0x2A8, "unsigned int")
 
-    async def card_spacing(self) -> int:
+    async def card_spacing_horizontal_adjust(self) -> int:
```

Comment:
Why

Reply:
looked at the offsets in CE  and the value and it made more sense of it being the horizontal card spacing, I also compared the DeckListControl and SpellListControl as they are similar in some aspects.

Reply:
The same function name seems to appear futher down as well though